lib/Main.js

```diff
@@ -486,8 +486,8 @@ Main.prototype.getOrCreateBridgedRoom = function(gitterName) {
         main: this,
         gitter: this._gitter,
         gitterRealtime: this._gitterRealtime,
-
         gitterRoomName: gitterName,
+        enablePresence: !(this._config.enable_presence === false)
```

Comment:
Why the `!(... === false)` and not just the thing?

Reply:
If missing we want it to be enabled by default. The only time we want it disabled is if it's explicitly set as false.

Reply:
It's probably a better idea to move that logic to the BridgedRoom constructor. And doing an explicit = enablePresence === undefined ? true : enablePresence

Reply:
Fair point, will do.